onlinejudge/_implementation/main.py

```diff
@@ -70,6 +70,7 @@ def get_parser() -> argparse.ArgumentParser:
     subparser.add_argument('-n', '--dry-run', action='store_true', help='don\'t write to files')
     subparser.add_argument('-a', '--system', action='store_true', help='download system testcases')
     subparser.add_argument('-s', '--silent', action='store_true')
+    subparser.add_argument('-t', '--yukicoder-token', type=str)
```

Comment:
```suggestion
    subparser.add_argument('--yukicoder-token', type=str)
```

これはかなり限定的だし verify helper みたいな wrapper スクリプトからしか使わないようなオプションです。手動でよければ普通にログインすればよい。なので、1文字オプションは割り当てたくない